upup/pkg/kutil/rollingupdate_cluster.go

```diff
@@ -266,8 +290,26 @@ func (n *CloudInstanceGroup) RollingUpdate(cloud fi.Cloud, force bool, interval
 			return fmt.Errorf("error deleting instance %q: %v", instanceID, err)
 		}
 
-		// TODO: Wait for node to appear back in k8s
+		// Wait for new EC2 instances to be created
 		time.Sleep(interval)
+
+		// Wait until the cluster is happy
+		// TODO: do we need to respect cloud only??
```

Comment:
Probably yes.... it's really important in the case when you have pushed a bad config and can't reach the API